sys/include/senml.h

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ * Copyright (C) 2016 Lennart Dührsen <lennart.duehrsen@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+/**
+ * @ingroup     senml
```

Comment:
oopsy, not yet. is it ok if i just put

`@defgroup senml Sensor Measurement List`

before the ingroup command?